src/macaron/config/defaults.py

```diff
@@ -163,9 +163,6 @@ def create_defaults(output_path: str, cwd_path: str) -> bool:
             os.path.relpath(os.path.join(output_path, "defaults.ini"), cwd_path),
         )
         return True
-    except shutil.Error as error:
+    except (OSError, shutil.Error) as error:
```

Comment:
Should we put a small comment above this line saying why we added the OSError here?

Reply:
See commit [dc4a44d](https://github.com/oracle/macaron/pull/383/commits/dc4a44dc54e47d3e09ac886fad768f327802da57)